src/integration-test/java/connection_check/ConnectionChecker.java

```diff
@@ -1,52 +0,0 @@
-
-
-package connection_check;
-
-import developerTC.AgentService;
-
-import java.net.InetSocketAddress;
-import java.net.Socket;
-import java.net.URL;
-
-public class ConnectionChecker {
-
-
-    private String uri;
-
-    public ConnectionChecker(String uri) {
-        this.uri = uri;
-    }
-
-    public boolean connect() {
-
-        return isServerReachable();
-    }
-
-    /**
-     * Checks HTTP URL and returns <code>true</code> a connection can be established
-     * to the corresponding host and port
-     *
-     * @return boolean if endpoint is up within the timeout
-     */
-    public boolean isServerReachable() {
```

Comment:
What is replacing this functionality? Is it being used at all? Should it be available as a utility function?